tests/pToken.js

```diff
@@ -133,4 +136,81 @@ contract("pToken", async accounts => {
     });
   })
 
+  describe('Claims Manager features', function () {
+    beforeEach(async function () {
+      claimsManager = await ClaimsManagerSingleAccount.new( {from: governance} )
+      targetpToken = await pToken.new(UnderlyingToken.address, governance, claimsManager.address)
+    });
+
+    it("should not allow a deposit if status is investigating", async () => {
+      underlyingToken = await UnderlyingToken.new( {from: governance} )
+
+      targetpToken = await pToken.new(underlyingToken.address, governance, claimsManager.address)
+      amount = new BN('30000000000000000000')
+      await underlyingToken.approve(
+        targetpToken.address,
+        amount,
+        { from: governance }
+      );
+      await claimsManager.setActivePayoutEvent(
+        true,{ from: governance }
+      );
+      await claimsManager.submitClaim(
+         { from: governance }
+      );
+      await expectRevert.unspecified(targetpToken.deposit(amount, { from: governance})
+      );
+    });
+
+    it("should allow a deposit if status is ready", async () => {
+      underlyingToken = await UnderlyingToken.new( {from: governance} )
+
+      targetpToken = await pToken.new(underlyingToken.address, governance, claimsManager.address)
+      amount = new BN('1000000000000000000')
+      await underlyingToken.approve(
+        targetpToken.address,
+        amount,
+        { from: governance }
+      );
+      await targetpToken.deposit(amount, { from: governance})
+
+      expect(await targetpToken.balance({from:governance})).to.be.bignumber.equal(amount);
+    });
+
+    /*
+        Below not working - unsure why
```

Comment:
@stephenbyrne99 is this still not working?

Reply:
nope